packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -18,27 +18,31 @@
     min-height: calc(var(--iui-size-m) * 2.5);
     height: 100%;
     cursor: pointer;
-    color: var(--iui-color-foreground-2);
+    color: var(--iui-color-text);
     transition: background-color var(--iui-duration-1) ease-out;
 
     &.iui-active {
-      color: var(--iui-color-foreground-primary);
+      color: var(--iui-color-text-primary);
 
       .iui-tab-icon {
-        fill: var(--iui-color-foreground-primary);
+        fill: var(--iui-color-icon-primary);
```

Comment:
This icon is standing out quite a bit, compared to the text and stripe. Is this the intended look?

![](https://user-images.githubusercontent.com/9084735/199602285-fcef80ec-e831-4cbe-b179-c2cc8ece0c74.png)

Reply:
Note to self:  use `currentColor` when icon is directly next to active text like this.

Reply:
what about other places like active tree nodes?

i feel like we could change the color value of `--iui-color-icon-accent` to be a little more consistent with `--iui-color-text-accent`, at least in dark theme where it stands out the most

Reply:
`dark.scss` has new `$fg-*-2` values.  How I created the new values:

Take `$fg-*-1` & current `$fg-*-2` value, plug them into this [middle color finder web app](https://www.dannybrien.com/middle/), and use that new value for `$fg-*-2`.  Here is the luminosity test:
![Screenshot 2022-11-07 at 1 41 30 PM](https://user-images.githubusercontent.com/849817/200389188-d4b41670-70fe-4ed8-86f7-15b0d8ba951d.png)